scss/styles.scss

```diff
@@ -8,7 +8,31 @@
   color: $gray;
 }
 
+.main-container {
+  display: flex;
+  justify-content: center;
+}
+
 .container {
+  display: flex;
+  flex-direction: column;
+}
+
+.back-button {
+  align-items: center;
+  display: flex;
+  font-size: 20px;
+  margin: 46px 0 108px;
```

Comment:
the top margin should be the parents padding as it is spacing a child from a parent